create-upgrade-symlinks.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 
 prev_versions=$(cat promscale.control | sed -n 's/# upgradeable_from = \(.*\)/\1/p' | sed "s/[[:space:]']//g" | tr ',' '\n')
-cur_version=$(cargo pkgid | cut -d'#' -f2 | cut -d':' -f2)
+# Note: we cut on both ':' and '@' here to support pre-1.62.0 and post 1.62.0 `cargo pkgid` output
+cur_version=$(cargo pkgid | cut -d'#' -f2 | cut -d':' -f2 | cut -d'@' -f2)
```

Comment:
Can we pass it as an arg from makefile to this script instead of repeating this?

Reply:
I've avoided repetition in a slightly different way than suggested.